tests/runner.py

```diff
@@ -663,9 +664,13 @@ def do_run(self, src, expected_output, args=[], output_nicerizer=None,
     for engine in js_engines:
       #print 'test in', engine
       js_output = self.run_generated_code(engine, filename + '.o.js', args, output_nicerizer=output_nicerizer, assert_returncode=assert_returncode)
+      js_output = js_output.replace('\r\n', '\n')
```

Comment:
`js_output.replace` was already called in `self.assertContained(expected_output, js_output.replace('\r\n', '\n'))`, I just moved it out of the condition to avoid minor code duplication.

The only difference with the old logic is that the search for 'ERROR' is now performed in the 'replaced' js_output, while initially it was done in the original js_output.